Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -530,15 +530,13 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             foreach (NodeModule module in modules) {
                 NodeModule newModule;
                 if (GetOrAddModule(module, out newModule)) {
-                    if (newModule.FileName != newModule.JavaScriptFileName) {
```

Comment:
I'm a bit more unsure what the original intention of this line was. I'm going to try stepping through it more with various node versions to see, but I never saw the old expression evaluate to `true` in my testing. 

My main concern is that the current approach may just be brute forcing breakpoints into working with Node 6, instead of addressing the fundamental issues. That could risk breaking both old and future node versions.